feature_tests/js/package.json

```diff
@@ -19,7 +19,6 @@
       "esm"
     ],
     "nodeArguments": [
-      "--experimental-wasm-bigint"
```

Comment:
I just noticed that this change made it into this PR; it is needed for Node 14 compatibility. I can move it to another PR if desired

Reply:
Have we changed CI to node 16 already? It should be in that PR

Reply:
Okay I removed this change from this PR so we don't have to figure it out here